sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/TextStyleName.java

```diff
@@ -13,6 +13,13 @@
  * Defines values for TextStyleName.
  */
 public final class TextStyleName extends ExpandableStringEnum<TextStyleName> {
+
+    /**
+     * Constructs a TextStyleName object
+     */
+    public TextStyleName() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.